compass_sdk/models/datasources.py

```diff
@@ -0,0 +1,50 @@
+# Python imports
+import datetime
+import typing
+
+# 3rd party imports
+import pydantic
+
+# NOTE: The models below are directly copied from the API.
+
+T = typing.TypeVar("T")
+
+Content: typing.TypeAlias = typing.Dict[str, typing.Any]
+
+
+class PaginatedList(pydantic.BaseModel, typing.Generic[T]):
+    value: typing.List[T]
+    nextPageToken: typing.Optional[str] = None
```

Comment:
remove this one, not supported yet and we likely support pagination using offsets and limit because it is easier for the end users.

Reply:
But this is what the API returns, so either we match the API, or we remove it from both.

Reply:
In the upcoming PR is removed in favor of skip and limit, so I would suggest remove it for now, it is not used, so if you ignore the field also if returned empty is safe.

Reply:
Removed `nextPageToken` field.